src/main/java/org/openstreetmap/atlas/geography/sharding/SlippyTile.java

```diff
@@ -65,6 +65,30 @@
      * @return All tiles within some bounds
      */
     public static Iterable<SlippyTile> allTiles(final int zoom, final Rectangle bounds)
+    {
+        if (zoom > MAX_ZOOM)
+        {
+            throw new CoreException("Zoom too large.");
```

Comment:
Nit: we should probably add the zoom here to make debugging a little easier, but this is super minor. Maybe you can update this in the other open PR, going to merge as is.